spec/populators/proceeding_type_populator_spec.rb

```diff
@@ -12,15 +12,15 @@
     let(:seed_file) { Rails.root.join("db/seed_data/proceeding_types.yml").freeze }
 
     it "create instances from the seed file" do
-      expect { described_class.call }.to change(ProceedingType, :count).by(61)
+      expect { described_class.call }.to change(ProceedingType, :count).by(159)
```

Comment:
I thought there were 95 new proceedings instead on 98?

Reply:
🤔 where are you getting the figure of 95 new proceedings from @naseberry ?

Reply:
The source spreadsheet looks a bit all over the place.
* where did the descriptions come from? The SS has PBM01 as `Declaration for overseas adoption-Pub Law-Fam`?
* the count on tab 1 does not match the count on the question matrix